app/views/vehicles/index.html.erb

```diff
@@ -1,6 +1,7 @@
 <h1>All Vehicles </h1>
 
 <% @vehicles.each do |vehicle| %>
+  <% next if vehicle.sold == true %>
```

Comment:
I took the solutions a different direction and added to the index method in the controller. This would be good to bring up at our check in.